qutebrowser/browser/browsertab.py

```diff
@@ -806,6 +808,10 @@ def __init__(self, mode_manager: modeman.ModeManager,
         self._tab = tab
         self._mode_manager = mode_manager
 
+    @abstractmethod
+    def __init_inspector(self):
```

Comment:
```suggestion
    def _init_inspector(self):
```

With the double underscore, you [turn on name mangling](https://docs.python.org/3/tutorial/classes.html#private-variables), making it impossible to override the class in subclasses.